src/utils/isEqual/index.ts

```diff
@@ -0,0 +1,33 @@
+/** *******************************************************************************************************************
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+  
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  
+      http://www.apache.org/licenses/LICENSE-2.0
+  
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.                                                                              *
+ ******************************************************************************************************************** */
+const isEqual = (arr1?: string[] | null, arr2?: string[] | null) => {
```

Comment:
Should we make it clearer what this function is doing by naming it e.g. `isEqualSet` or `arrayContainSameElements`?

Reply:
I think the function name + argument signature should tell the fact: is to compare two string array to see whether they are equal?

Reply:
it's not obvious from the signature that it sorts the arrays before comparing them